internal/api/migrate/loader.go

```diff
@@ -65,13 +65,15 @@ func (c *migCuePart) GetSchemaPath() string {
 	return c.schemasPath
 }
 
-func (c *migCuePart) Load() error {
-	conditions, err := c.buildListOfConditions()
+// NB There are no relevant case of "failed loads" in the case of migration schemas at the moment
+func (c *migCuePart) Load() (successfulLoadsCount, _ int, err error) {
```

Comment:
doc should start with Load :D 
this can be the opportunity to tell what this function is doing before adding a NB :P
```
// Load the migration schema ?
// NB: ...
```